front/components/ui/Layout.tsx

```diff
@@ -0,0 +1,13 @@
+import { Box } from "@chakra-ui/react";
```

Comment:
nie potrzebujemy katalogu ui. wszystko co jest we front/components to już jest ui :)

Reply:
To ja jestem szczerze przyzwyczajony do takiej konwecji, że jest golder UI, który ma takie ogólne rzeczy w sobie, ale możemy się pozbyć i dać to gdzieś indziej tylko nie mam pomysłu na inną nazwę :)